### PR TITLE
theming and animations

### PR DESCRIPTION
**Pull request by naveenmadur**
## Description
- changed theme and added animations

### CODE REVIEW

File: lib/core/app_theme/colors_n_theme/app_themes.dart [hunk 1]
```diff
@@ -80,37 +95,56 @@ ThemeData getThemeFor(
...
-      color: AppColors.cF1F1FA,
+      color: AppColors.white,
     ),
-    borderRadius: BorderRadius.circular(16),
+    borderRadius: BorderRadius.circular(20),
   ),
   enabledBorder: OutlineInputBorder(
     borderSide: const BorderSide(
-      color: AppColors.cF1F1FA,
+      color: AppColors.white,
```
Comment (akanksha-coditas) [inline lib/core/app_theme/colors_n_theme/app_themes.dart:134]: Don't we need any default border in containers?

File: lib/features/authentication/presentation/pages/login_screen.dart [hunk 2]
```diff
@@ -45,15 +45,14 @@ class _LoginScreenState extends State<LoginScreen> {
...
                 return login(context);
               } else if (state is LoadingState) {
                 return const LoadingContainer();
               } else if (state is ErrorState) {
                 return const ErrorContainer();
               } else if (state is LoggedIn) {
-                // TODO(naveen): migrate to go
                 WidgetsBinding.instance.addPostFrameCallback((_) {
                   context.go('/chatHomePage');
                 });
```
Comment (akanksha-coditas) [inline lib/features/authentication/presentation/pages/login_screen.dart:58]: Remove this commented code as we are not using autorouters.

File: lib/features/authentication/presentation/pages/login_screen.dart [hunk 3]
```diff
@@ -66,8 +65,14 @@ class _LoginScreenState extends State<LoginScreen> {
                 //     ),
                 //   ),
                 // );
+
+                if (state.account?.id == '112315523802286799800') {
```
Comment (akanksha-coditas) [inline lib/features/authentication/presentation/pages/login_screen.dart:69]: Why are we using this Id?

File: lib/features/chat_module/presentation/pages/chat_home_page.dart [hunk 4]
```diff
@@ -55,19 +59,30 @@ class ChatHomePageState extends State<ChatHomePage>
...
-              controller: _tabController,
-              tabs: const <Widget>[
-                Tab(
-                  text: 'CHATS',
-                ),
-                Tab(
-                  text: 'CALLS',
+            DecoratedBox(
+              decoration: const BoxDecoration(
+                color: AppColors.white,
```
Comment (akanksha-coditas) [inline lib/features/chat_module/presentation/pages/chat_home_page.dart:67]: We have already given background color. So I don't think we need to specify color explicitly.

File: lib/features/chat_module/presentation/pages/chat_list/call_list_screen.dart [hunk 5]
```diff
@@ -57,10 +54,11 @@ class CallListScreen extends StatelessWidget {
               if (snapshot.data?.isEmpty ?? true) {
                 return const InfoContainer('No data found');
               } else {
-                return SingleMessage(
-                  snapshot.data![index],
-                  '',
-                );
+                return const SizedBox();
+                // return SingleMessage(
```
Comment (akanksha-coditas) [inline lib/features/chat_module/presentation/pages/chat_list/call_list_screen.dart:58]: Remove commented code if not required.

File: lib/features/chat_module/presentation/pages/chat_list/chat_list_screen.dart [hunk 6]
```diff
@@ -569,13 +588,25 @@ class _ChatListScreenState extends State<ChatListScreen> {
...
               ),
             ),
           ),
-        );
+        ).animate().moveX(
+              begin: -30,
+              end: 0,
+              duration: 70.ms,
+              curve: Curves.decelerate,
+              // begin: Offset.zero,
```
Comment (akanksha-coditas) [inline lib/features/chat_module/presentation/pages/chat_list/chat_list_screen.dart:607]: Remove if not required.

File: lib/features/chat_module/presentation/pages/group_chat/widgets/custom_popup_menu_button.dart [hunk 7]
```diff
@@ -1,6 +1,7 @@
 import 'package:coditas_chat/core/app_theme/app_theme.dart';
 import 'package:coditas_chat/injection_container.dart';
 import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
 import 'package:google_sign_in/google_sign_in.dart';
 
 class CustomPopUpMenuButton extends StatelessWidget {
```
Comment (akanksha-coditas) [inline lib/features/chat_module/presentation/pages/group_chat/widgets/custom_popup_menu_button.dart:7]: Do we need this? As we have drawer.

Comment (naveenmadur) [reply lib/features/chat_module/presentation/pages/group_chat/widgets/custom_popup_menu_button.dart:7]: Discussed with @SumitAmbhoreCoditas to remove drawer and user popUp menu button.

File: lib/core/app_theme/colors_n_theme/app_themes.dart (hunk 1 above)
Comment (naveenmadur) [reply lib/core/app_theme/colors_n_theme/app_themes.dart:134]: No. We have decided to remove border

File: lib/features/chat_module/presentation/pages/chat_list/call_list_screen.dart (hunk 5 above)
Comment (naveenmadur) [reply lib/features/chat_module/presentation/pages/chat_list/call_list_screen.dart:58]: Removed

File: lib/features/chat_module/presentation/pages/chat_list/chat_list_screen.dart (hunk 6 above)
Comment (naveenmadur) [reply lib/features/chat_module/presentation/pages/chat_list/chat_list_screen.dart:607]: Removed

File: lib/features/chat_module/presentation/pages/chat_home_page.dart (hunk 4 above)
Comment (naveenmadur) [reply lib/features/chat_module/presentation/pages/chat_home_page.dart:67]: Done

File: lib/features/chat_module/presentation/pages/group_chat/widgets/custom_popup_menu_button.dart (hunk 7 above)
Comment (naveenmadur) [reply lib/features/chat_module/presentation/pages/group_chat/widgets/custom_popup_menu_button.dart:7]: No drawer only popUp menu

**Pull request merged**